config/config.go

```diff
@@ -228,6 +228,14 @@ type ScrapeConfig struct {
 	Scheme string `yaml:"scheme,omitempty"`
 	// The HTTP basic authentication credentials for the targets.
 	BasicAuth *BasicAuth `yaml:"basic_auth,omitempty"`
+	// The bearer token for the targets.
+	BearerToken string `yaml:"bearer_token,omitempty"`
```

Comment:
Are they _necessarily_ mutually exclusive? I having both a client cert and a login may not be very useful but I don't see a reason to forbid it. It's on the server to deal with this.